index.js

```diff
@@ -1,8 +1,59 @@
 const express = require('express')
+const app = express()
 const path = require('path')
 const PORT = process.env.PORT || 5000
-const app = express()
+const bodyParser = require('body-parser')
+const db = require("./server/db.js");
 
-app.use(express.static(path.join(__dirname, 'assets')))
-app.get('/', (req, res) => res.sendFile(path.join(__dirname,'assets/html/home.html')))
+app.use(bodyParser.urlencoded({ extended: false}))
+app.use(bodyParser.json())
+app.use(express.static(path.join(__dirname, 'client')))
+app.get('/', (req, res) => res.sendFile(path.join(__dirname,'client/html/home.html')))
 app.listen(PORT, () => console.log(`Listening on ${ PORT }...`))
+
+db.initialise;
+app.post('/signin', function (req, res) {
+  // Search a record
+  const name = req.body.user_name;
+  const pw = req.body.password;
+  const cb = function (err, row) {
+    if (req.body.user_name) {
+      if (!err) {
+        if (row) {
+          res.status(200).send({ exsisting: true });
+        } else {
+          console.log("No record matches")
+          res.status(401).send({ exsisting: false });
+        }
+      } else {
+        console.log(err)
```

Comment:
For stylistic purposes, use `console.error(err)` instead to make the errors pop out more on your terminal and in your logs

Reply:
Changed!